infrastructure/templates/public-api/application/shared/appGateway.bicep

```diff
@@ -51,3 +54,12 @@ module appGatewayModule '../../components/appGateway.bicep' = {
     tagValues: tagValues
   }
 }
+
+module backendPoolsHealthAlert '../../components/alerts/appGateways/backendPoolHealth.bicep' = if (deployAlerts) {
+  name: '${resourceNames.sharedResources.appGateway}BackedPoolsHealthDeploy'
```

Comment:
There's a typo here `Backed/Backend`.

Also, should 'Pools' be plural?

Reply:
Ooh good spot - corrected thanks.  Pools should be plural cos the gateway has multiple backend pools that this single alert covers ta :)